doc/2/framework/classes/backend-import/users/index.md

```diff
@@ -0,0 +1,39 @@
+---
+code: true
+type: page
+title: users
+description: BackendImport.users method
+---
+
+# `users()`
+
+<SinceBadge version="auto-version" />
+<CustomBadge type="error" text="Experimental: non-backward compatible changes or removal may occur in any future release."/>
+
+Loads users into the app.
+
+This method is idempotent. If a user is defined multiple times before the app starts with the same kuid, only the last one will be retained.
+
+::: info
+This method can only be used before the application is started.
+:::
+
+```ts
+users(users: JSONObject): void
```

Comment:
Missing `onExistingUsers` option, also it should be an object like `users(users, { onExistingUsers: 'fail' })`